ragas/src/ragas/metrics/_factual_correctness.py

```diff
@@ -287,5 +293,13 @@ async def _single_turn_ascore(
 
         return np.round(score, 2)
 
+    async def decompose_and_verify_claims(
```

Comment:
Can we name this something more generic for clarity.

How about `get_claim_verification_results` ?